spotlight-client/src/contentApi/sources/us_me.ts

```diff
@@ -0,0 +1,685 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { TenantContent } from "../types";
+import { ageGroup, gender, raceOrEthnicity } from "./methodologyBoilerplate";
+import programRegionsTopology from "./usNdProgramRegions";
+
+// localities for both sentencing and probation
+const judicialDistricts = [
```

Comment:
What is the process for updating the sources file for a new state?

Is there an authoritative source for correctness?

Reply:
Ideally this has already been created somewhere within the data platform as part of the ingest setup. In the past there has not really been a clear and obvious source for it, I've just had to hunt around for it. Now that we have State Pods I would say you should ask Kris or Daniela (I don't know how much of the ME ingest she has handed off yet)

Reply:
When I did this for ID/TN I ran this query in BQ and just used those:

```
SELECT supervision_type, district
FROM `recidiviz-staging.public_dashboard_views.supervision_population_by_district_by_demographics`
WHERE state_code = 'US_ME'
group by supervision_type, district
order by supervision_type, district;
```

It looks like for probation they aren't hydrated yet, but for parole there are some that you could use.

Reply:
I don't think I have bigquery access to this project? What are you getting?

Reply:
Oh oops, here are the results!

<img width="838" alt="Screen Shot 2022-05-06 at 3 15 38 PM" src="https://user-images.githubusercontent.com/3762913/167222981-54038b6a-98d5-4229-b366-d148be80efa6.png">

Reply:
I've added those in before, however I find it strange that the data gives me the same numbers for both "ALL" and "EXTERNAL_UNKNOWN", and I think Maine doesn't have parole, and district breakdowns are all in the "PAROLE" category...

Reply:
The end result being in the Sentencing by district and Probation by district sections, "All" and "Unknown" gives me the same result, and selecting a district gives me "No Data".

Reply:
Fetching `sentence_type_by_district_by_demographics` gives me no district breakdowns, and fetching `supervision_population_by_district_by_demographics` gives me district breakdowns only in the Parole category. Is this something for the implementation engineer to make sure all the data is available?

Reply:
@terryttsai thanks for calling this out, these are great questions. Can you upload screenshots or videos of this behavior?

@hobuobi maybe you can help here? Or we can involve Kris/Daniela?

Reply:
<img width="1530" alt="Screen Shot 2022-05-09 at 1 02 04 PM" src="https://user-images.githubusercontent.com/5040739/167488412-fa8b8d48-5137-49a2-a905-244f965b5d23.png">
<img width="1529" alt="Screen Shot 2022-05-09 at 1 02 11 PM" src="https://user-images.githubusercontent.com/5040739/167488417-e112e4ac-f714-4a91-ac7c-740ee6478a4e.png">
<img width="1528" alt="Screen Shot 2022-05-09 at 1 02 22 PM" src="https://user-images.githubusercontent.com/5040739/167488421-3e3771dc-bbd2-4bef-b947-9b8852093f72.png">